src/test/java/org/openmrs/performance/http/DoctorHttpService.java

```diff
@@ -261,6 +261,16 @@ public HttpRequestActionBuilder getImmunizations(String patientUuid) {
 				.get("/openmrs/ws/fhir2/R4/Immunization?patient=" + patientUuid + "&_summary=data");
 	}
 	
+	public HttpRequestActionBuilder getPrograms() {
+		return http("Get Programs of Patient")
```

Comment:
```suggestion
		return http("Get Programs")
```
This endpoint is to get all programs, if we get anything patient-specific, we need to send the patient uuid.

Reply:
Got the point!